{
  "Header": {
    "Original": "CVE-2019-3857 (An integer overflow flaw which could lead to an out of bounds write wa ...)",
    "Line": 315805,
    "ID": "CVE-2019-3857",
    "Description": "(An integer overflow flaw which could lead to an out of bounds write wa ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-4431-1 DLA-1730-1}",
      "Line": 315806,
      "Type": "xref",
      "Bugs": [
        "DSA-4431-1",
        "DLA-1730-1"
      ]
    },
    {
      "Original": "- libssh2 1.8.0-2.1 (bug #924965)",
      "Line": 315807,
      "Type": "package",
      "Package": "libssh2",
      "Kind": "fixed",
      "Version": "1.8.0-2.1",
      "BugNo": 924965
    },
    {
      "Original": "NOTE: https://www.libssh2.org/CVE-2019-3857.html",
      "Line": 315808,
      "Type": "NOTE",
      "Description": "https://www.libssh2.org/CVE-2019-3857.html"
    },
    {
      "Original": "NOTE: Patch: https://libssh2.org/1.8.0-CVE/CVE-2019-3857.patch",
      "Line": 315809,
      "Type": "NOTE",
      "Description": "Patch: https://libssh2.org/1.8.0-CVE/CVE-2019-3857.patch"
    },
    {
      "Original": "NOTE: https://github.com/libssh2/libssh2/pull/315",
      "Line": 315810,
      "Type": "NOTE",
      "Description": "https://github.com/libssh2/libssh2/pull/315"
    }
  ]
}